src/hooks/useDeleter.js

```diff
@@ -0,0 +1,17 @@
+import { useState } from "react";
+import { deleteComment } from "../api";
+
+const useDeleter = (comment_id) => {
```

Comment:
This would be better named as just "deleter" - it should be named after what it does, not how we are using it.